packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -1,53 +0,0 @@
-#!/bin/bash
```

Comment:
This script is no longer necessary because:
1. We are moving the logic the `gutenberg-mobile` repository
2. Some of the logic is covered by `publish-to-s3` plugin
3. We are simplifying this process to address some developer frustrations